crates/spark/src/signer/mod.rs

```diff
@@ -20,6 +21,12 @@ pub trait Signer: Send + Sync + 'static {
         message: &[u8],
     ) -> Result<Signature, SignerError>;
 
+    fn sign_message_ecdsa_recoverable_from_path(
+        &self,
+        message: &[u8],
+        derivation_path: &DerivationPath,
+    ) -> Result<RecoverableSignature, SignerError>;
+
```

Comment:
Wouldn't it make sense to create a new signer trait on the Breez SDK level that implements the Spark signer methods and these new ones? Then spark crate users wouldn't need to implement these and Breez SDK users would still have a single trait to implement.

Reply:
Yes, that makes sense to have a signer trait on the Breez SDK level that contains the additional functions. Then create an implementation of the spark signer in the Breez SDK that pipes everything to the SDK signer. It's definitely the cleaner approach. I'll do that. Agreed @roeierez?